### PR TITLE
Adding our contribution agreement and doing a little polish

### PR DESCRIPTION
**Pull request by robblau**
First pass at adding in a page with instructions on how to contribute to the documentation repo and links to the agreements that people will need to sign to be able to commit.

Also applied a little polish to some of the wording and the landing page logos.

### CODE REVIEW

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/f5af9b311e98fa04eafcca301df59f120af95857/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/bf0e603a53f64d3885411030724c9fbea5e538c0/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/c3a6e8cdebf3104cf90d8353a6deb14ef01d81bc/index.html)

**Comment by robblau** [comment]
@manneohrstrom what would be needed to make the boxes around the logo and copy on the landing page to grow correctly with its content?

**Comment by manneohrstrom** [comment]
@robblau i think that's a little tricky - but i can look into it - i think easiest thing is just to make them bigger.

**Comment by manneohrstrom** [comment]
One more thought - should we also add the content related contribution items we discussed?

## Content Authoring guidelines

### Diagrams and figures
- Use https://www.draw.io/ to author diagrams.
- The source file should be saved next to the image.
- Open sans (https://fonts.google.com/specimen/Open+Sans) should be used for text.
- We are not translating any images, so text should be kept to a minimum.
- Exported images should be 144dpi and stored in png format.
- Graphics should be functional in style, typically using grayscales only.
 
### Screenshots
- Screenshots should be 144dpi and stored in png format
- Post processing should be kept to a minimum
- If possible, screenshot an entire window rather than cropping things
    - Use CMD+SHIFT+4 on the mac for easy screenshots of windows

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/0137bf33ae951b4e696236d5ae23534775becbaa/index.html)

**Comment by robblau** [comment]
Addressed @manneohrstrom 's comments. @tannazie any comments on content? It would be nice to merge anything for translation quickly, even if the boxes still need to be adjusted. Ok by you two?

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/de85c3138e07c899d26693174249e72d0303819a/index.html)

**Comment by tannazie** [comment]
Content all looks great to me. My only nitpick is to get rid of the the last comma here:

`The Toolkit Core API is the platform upon which all integrations are built. The Core API handles app, engine, and platform fundamentals, as well as file system organization, and tool deployment.`

(This is in `toolkit.md` – if there's a better to attach a comment to a specific file or line, let me know.)

File: docs/en/toolkit.md [hunk 1]
```diff
@@ -8,6 +8,6 @@ has_children: true
 
 # Toolkit
 
-Toolkit is an extendable platform for integrating artist applications and pipeline tools with Shotgun.
+Toolkit is an extensible platform for integrating artist applications and pipeline tools with Shotgun.
 
-The Toolkit Core API is the platform upon which all integrations are built. The Core API handles app, engine, and platform fundamentals, as well as file system and deploy management.
+The Toolkit Core API is the platform upon which all integrations are built. The Core API handles app, engine, and platform fundamentals, as well as file system organization, and tool deployment.
```
Comment (robblau) [inline docs/en/toolkit.md]: @tannazie from the "Files Changed" tab you can actually go down to the markdown itself and just click the "+" next to the line you want to comment on and the comment will be associated with that line.

Will fix.

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/f35cd556767c560911f59b9ebde7d64b187ad485/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/276727986385476e53d9c6ad47bc9109c5f1ef53/index.html)

**Pull request merged**